packages/babel-plugin-transform-async-to-generator/test/fixtures/regression/7178/output.js

```diff
@@ -1,16 +1,16 @@
+var _Contact;
+
 const title = "Neem contact op";
 
 function action() {
   return _action.apply(this, arguments);
 }
 
-var _ref = /*#__PURE__*/React.createElement(Contact, {
-  title: title
-});
-
 function _action() {
   _action = babelHelpers.asyncToGenerator(function* () {
-    return _ref;
+    return _Contact || (_Contact = /*#__PURE__*/React.createElement(Contact, {
```

Comment:
In Babel 8 we could just use `||=` here, and if someone targets an older browser it will be transformed by another different plugin.